packages/web3/package-lock.json

```diff
@@ -0,0 +1,10235 @@
+{
```

Comment:
This `package-lock.json` file is new, is it necessary?

Reply:
No it is not